github/ipld.yml

```
@@ -3155,18 +3109,10 @@ teams:
         - rvagg
         - Stebalien
         - vmx
-        - warpfork
       member:
         - BigLep
```

```suggestion
        - aschmahmann 
```
Swap Adin for me as an IP Shipyard representative.